assemblies/DataSources/OpenConfig/Skyline.DataMiner.DataSources.OpenConfig.Gnmi.Protocol.xml

```diff
@@ -204,67 +300,265 @@
+            <summary>
+            Tries to get the JSON token values for the requested path filter.
+            </summary>
+            <param name="entityPath">Path filter in the data grid entity that specifies what JSON token value should be returned for a specific level.</param>
+            <param name="pathKeyParts">Parts of the primary key.</param>
+            <param name="token">JSON token to be investigated.</param>
+            <param name="pathLevel">Location of the token value in the path filter.</param>
+            <param name="rowIndex">In case the direct parent is an array, it will contain the index of the token of that array.</param>
+            <param name="keysAndTokens">Found JSON token values with their key parts.</param>
+            <returns>Boolean indicating if token values were found in the requested path.</returns>
+        </member>
+        <member name="M:Skyline.DataMiner.DataSources.OpenConfig.Gnmi.Protocol.DataMapper.DataMinerConnectorDataGrid.TryGetJsonTokenForPath(Gnmi.Path,System.Collections.Generic.List{Skyline.DataMiner.DataSources.OpenConfig.Gnmi.Utils.PathKeyPart},Newtonsoft.Json.Linq.JArray,System.Int32,System.Collections.Generic.List{Skyline.DataMiner.DataSources.OpenConfig.Gnmi.Protocol.DataMapper.DataMinerConnectorDataGrid.KeyAndToken})">
+            <summary>
+            Tries to get the JSON token values for the requested path filter.
+            </summary>
+            <param name="entityPath">Path filter in the data grid entity that specifies what JSON token value should be returned for a specific level.</param>
+            <param name="pathKeyParts">Parts of the primary key.</param>
+            <param name="token">JSON array to be investigated.</param>
+            <param name="pathLevel">Location of the token value in the path filter.</param>
+            <param name="keysAndTokens">Found JSON token values with their key parts.</param>
+            <returns>Boolean indicating if token values were found in the requested path.</returns>
+        </member>
+        <member name="M:Skyline.DataMiner.DataSources.OpenConfig.Gnmi.Protocol.DataMapper.DataMinerConnectorDataGrid.TryGetJsonTokenForPath(Gnmi.Path,Newtonsoft.Json.Linq.JToken,System.Int32,Newtonsoft.Json.Linq.JToken@)">
+            <summary>
+            Tries to get a JSON token value on the specified path.
+            </summary>
+            <param name="path">Path for which the token is requested.</param>
+            <param name="token">JSON token to be investigated.</param>
+            <param name="pathLevel">Location of the token value in the path.</param>
+            <param name="foundToken">JSON token value on the requested path.</param>
```

Comment:
Is the token actually part of the path, or can it be found in a location indicated by the path? This because the preposition  use (in/on) is inconsistent and may not be correct, but the correct preposition depends on where the token is actually located. (This also occurs in other locations in this file.)